llama-index-integrations/llms/llama-index-llms-zhipuai/README.md

```diff
@@ -11,13 +11,13 @@
 
 ```py
 # Import ZhipuAI
-from llama_index.llms.zhipuai import ZhipuAI
+from llama_index.llms.zhipuai import ZhipuAILLM
```

Comment:
I think the convention that most have adopted would be to use what you have already:
- `ZiphuAI` for the LLM and `ZiphuAIEmbedding` for the embedding.

Reply:
For example:

- for LLMs: `Cohere`, `OpenAI`, `AlephAleph`, `Databricks` 
- for embeddings: `CohereEmbedding`, `OpenAIEmbedding`, `AlephAlephEmbedding`, and `DataBricksEmbedding`

However, with that being said there are a minority of cases that use the naming convention you are suggesting to switch to here. I'll leave it up to you.

Reply:
thanks for your reminder and advice.